cibyl/models/ci/system.py

```diff
@@ -0,0 +1,55 @@
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from cibyl.models.attribute import AttributeValue, AttributeListValue
+
+
+class System:
+    """
+        General model for a CI system. Holds basic information such as its
+        name, type and which pipelines and jobs it has.
+    """
+
+    def __init__(self, name: str, type_system: str):
+        self.name = AttributeValue(name="name", type=str, value=name,
+                                   arguments=['--system-name'])
```

Comment:
We'll have to change it once PR #10 is merged to "arguments=[Argument(name='--system-name', ...)]"